### ISSUE TITLE
-b <BYTESTRING> adds an extra "quit"

### ISSUE THREAD

**Issue by heasm66** [cspiegel/zdevtools#3]
`zd -b "c1 8f ff 03 eb 4d"` decompiles to: 
```
MAIN ROUTINE 0
     0: je               Gef #3eb [FALSE] 11
     6: quit
```
Why the `quit` that's not part of the bytestring?

**Comment by cspiegel** [comment]
The bytestring option was bolted on after the normal disassembler had been written, so it is constrained by how that one operates.

The normal (story file) disassembler assumes that the story will never just fall off the end of the file, i.e. it assumes there will _always_ be a quit instruction somewhere.  Because the bytestring option was an afterthought, the same restriction applies to it as well, so a quit instruction is tacked on to prevent failure.  That was a quick and dirty way of getting bytestrings working with not too much effort.  To be honest I never really figured anybody but me would ever use this feature...

So, that being said, I'm certainly in favor of improving the bytestring feature to not tack on that quit instruction, but it's not a trivial thing to do.  I'll try to figure out a good way to do it without disrupting things too much.

The man page _does_ document this behavior, although it fails to provide a rationale for it.

**Comment by cspiegel** [comment]
Ok, it wasn't trivial, but it wasn't really hard.  I've got a change in which will no longer add a `quit` to bytestrings.

**Comment by cspiegel** [comment]
Fixed in 2.1.0.
